.github/workflows/cleanup.yml

```diff
@@ -31,8 +31,31 @@ jobs:
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TOOL_TOKEN }}"
           PROJ_TOOLS=e1e498-tools PROJ_DEV=e1e498-dev bash openshift/scripts/oc_cleanup_tools.sh ${SUFFIX} apply
 
-      - name: Clean up database (OCP4)
+      - name: Clean up api - test (OCP4)
```

Comment:
I think that oc_cleanup_test.sh has the potential to become a complete cut and paste of oc_cleanup_dev.sh ; I think we should have a single oc_cleanup.sh (maybe with a better name?), where you specify the target as either dev or test.

It's true that right now test is a subset of dev - but if we have a single thing, we can be sure it will clean up both correctly.